git-ftp

```diff
@@ -478,7 +478,7 @@ upload_file() {
 upload_file_buffered() {
 	local SRC_FILE="$1"
 	local DEST_FILE="${SRC_FILE#$SYNCROOT}"
-	local ENC_DEST_FILE="${DEST_FILE//#/%23}"
+	local ENC_DEST_FILE="$(echo "${DEST_FILE//#/%23}" | sed 's/ /%20/g')"
```

Comment:
Cool, that should work. But the previous syntax `"${DEST_FILE//#/%23}"` was already replacing one character. It would probably be better if we used the same technique.